sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java

```diff
@@ -112,8 +131,31 @@ public HttpPipeline getHttpPipeline() {
             });
     }
 
-    // TODO: This is a temporary implementation for sample purposes. This should be spruced up/replaced once this API is actually designed.
-    // Input is Object and output is Response<T>.
+    /**
+     * Creates a digital twin.
+     *
+     * @param digitalTwinId The Id of the digital twin.
+     * @param digitalTwin The application/json digital twin to create.
+     * @param clazz The model class to convert the response to.
```

Comment:
"The model class to deserialize the response to"

Reply:
I think we should create a task on the board to go through all doc comments and ensure they are in sync.

Reply:
If that is an attempt to not try to do it right now, I'd advise not to do that. You'll create a monster and unpleasant work item for someone later.

Try to get it as right as possible now. A task later would be to sanity check it and all ensure consistency.

Reply:
@timtay-microsoft this method isn't doing both serialization and deserialization, so why mention both?

Reply:
Yes, I meant the 2nd option, since we already have a bunch of implementations (with javadoc comments) merged in, which used a different wording.

Reply:
Sure. I don't want to lower the bar for initial commits for comments. They should be held to the same standard as the code, at least. Agreed?

Reply:
In the create method, we serialize their parameter to send it over the wire, and then we deserialize the http response payload into the same type. But yeah, we can circle back on documentation consistency later

Reply:
@timtay-microsoft you are right. I'm on board.

Reply:
We only desiralize the response as per the "clazz". digital Twin input is Object. Am I missing something?